src/app.js

```diff
@@ -1 +1,43 @@
 'use strict';
+
+const { createNumber } = require('./createNumber');
+const { calculateBullsCows } = require('./bullsAndCows');
+const { myInterface } = require('./myInterface');
+
+const gameNumberLength = 4;
```

Comment:
Number 4 is present in many places, if you decided to make a variable for it, then use it in all places, and be consistent.